quill-sql/src/main/scala/io/getquill/source/sql/norm/MergeSecondaryJoin.scala

```diff
@@ -0,0 +1,30 @@
+package io.getquill.source.sql.norm
+
+import io.getquill.ast._
+
+object MergeSecondaryJoin extends StatelessTransformer {
```

Comment:
Could you clarify the purpose of this transformation? How does it relate to `ExpandJoin`?